### ISSUE TITLE
Mic Level drops to 0 when MyCroft is transcribing.  But fine during audio streamings. (cannot "Barge-in")

### ISSUE THREAD

**Issue by iointerrupt** [MycroftAI/mycroft-core#3005]
I spent some time tweaking mycroft and pulseaudo echo cancellation to be able to hear my audio (to some degree) so that I can issue commands like "hey mycroft... stop" while listening to music "Play Pandora" or listen to the news "play the news".  However any command that gets issued to MyCroft where mycroft TTS itself is speaking (utterance), I cannot barge in.  I use the mycroft-cli-client to monitor the mic levels, its working fine as it goes up and down depending on the volume during streaming.  However during utterance where mycroft is transcribing, mic level is 0 so I am unable to barge in.

**To Reproduce**
1. Go to 'Launch mycroft-cli-client'
2. Say 'who is Tom Hanks' (long enough text)
3. Mic-Level is 0 while MyCroft is transcribing

**Expected behavior**
The mic level should continue to work

**Environment (please complete the following information):**
 - Device type: Raspberry Pi4 4GB
 - OS: Picroft
 - Mycroft-core version: 21.2.1
 - Speaker/Mic: Jabra Speak 410

**Comment by forslund** [comment]
Just checking, have you set the [mute_during_output](https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/configuration/mycroft.conf#L189) listener config to false?

if you haven't you can add a section to your user config

`mycroft-config edit user`

and then add a section for the listener (or extend an existing one)

```
  "listener": {
    "mute_during_output": false
  }
```

**Comment by iointerrupt** [comment]
I think I figured out the issue:

As JSON standard dictates, the booleans should be false (w/o) quotes.  The problem appears to be that when you use mycroft-config set it will wrap boolean values in quotes.  I used mycroft-config set with `mycroft-config set listener.mute_during_output false` which automatically wrapped the value in quotes.  This might be one thing to be aware of.

As I noted in a previous Issue #2991 , the `mycroft-config set `is currently broken as its referencing the old myconf file.  I applied a fix for it on my local instance of picroft.  This might be something to watch out for when utilizing mycroft-config script to set boolean values.
